tests/snuba/api/endpoints/test_organization_events_span_indexed.py

```diff
@@ -1038,3 +1044,10 @@ def test_skip_aggregate_conditions_option(self):
             },
         ]
         assert meta["dataset"] == self.dataset
+
+
+class OrganizationEventsEAPRPCSpanEndpointTest(OrganizationEventsEAPSpanEndpointTest):
+    """These tests aren't fully passing yet, currently inheriting xfail from the eap tests"""
```

Comment:
Are the eap tests passing? If so, can we re-enable them and xfail these directly?

Reply:
... i'm scared to, but we can try re-enabling for a bit to see how stable things are